Lib/fontTools/feaLib/parser.py

```diff
@@ -273,6 +294,8 @@ def split_glyph_range_(self, name, location):
                 location)
 
     def parse_glyphclass_(self, accept_glyphname):
+        """Parses a glyph class, either named or anonymous, or (if
+        ``accept_glyphname==True``) a glyph name."""
```

Comment:
`accept_glyphname==True` is more specific than the implementation requires. The real requirement is `bool(accept_glyphname)`.